src/test/scala/org/broadinstitute/dsde/firecloud/EntityServiceSpec.scala

```diff
@@ -111,62 +124,69 @@ class EntityServiceSpec extends BaseServiceSpec with BeforeAndAfterEach {
+          // the "new MockRawlsDAO()" here is only to get access to a pre-canned WorkspaceResponse object
+          val workspaceResponse = new MockRawlsDAO().rawlsWorkspaceResponseWithAttributes
 
-            when(cwdsDAO.isEnabled).thenReturn(true)
-            when(cwdsDAO.getSupportedFormats).thenReturn(List("pfb","tdrexport", "rawlsjson"))
-            when(importServiceDAO.importJob(any[String], any[String], any[AsyncImportRequest], any[Boolean])(any[UserInfo]))
-              .thenReturn(Future.successful(RequestComplete(StatusCodes.Accepted, "")))
-            when(rawlsDAO.getWorkspace(any[String], any[String])(any[UserInfo]))
-              .thenReturn(Future.successful(workspaceResponse))
+          when(cwdsDAO.isEnabled).thenReturn(true)
+          when(cwdsDAO.getSupportedFormats).thenReturn(List("pfb","tdrexport", "rawlsjson"))
+          when(rawlsDAO.getWorkspace(any[String], any[String])(any[UserInfo]))
+            .thenReturn(Future.successful(workspaceResponse))
 
-            entityService.importEntitiesFromTSV("workspaceNamespace", "workspaceName", tsvData, dummyUserInfo("token"), true).futureValue
+          entityService.importEntitiesFromTSV("workspaceNamespace", "workspaceName", tsvData, dummyUserInfo("token"), true).futureValue
 
           val argumentCaptor = ArgumentCaptor.forClass(classTag[AsyncImportRequest].runtimeClass).asInstanceOf[ArgumentCaptor[AsyncImportRequest]]
 
           verify(cwdsDAO, times(1))
             .importV1(any[String], argumentCaptor.capture())(any[UserInfo])
           val capturedRequest = argumentCaptor.getValue
           capturedRequest.options should be(Some(ImportOptions(None, Some(tsvType != "update"))))
-          verify(importServiceDAO, never)
-              .importJob(any[String], any[String], any[AsyncImportRequest], any[Boolean])(any[UserInfo])
           verify(rawlsDAO, times(1))
-              .getWorkspace(any[String], any[String])(any[UserInfo])
+            .getWorkspace(any[String], any[String])(any[UserInfo])
 
-          }
+        }
     }
 
     "should return error for (async=true) when failed to write to GCS" in {
       val testGoogleDAO = new ErroringGoogleServicesDAO
       val entityService = getEntityService(mockGoogleServicesDAO = testGoogleDAO)
-      val caught = intercept[StorageException] {
+      val response =
         entityService.importEntitiesFromTSV("workspaceNamespace", "workspaceName",
           tsvParticipants, userToken, isAsync = true).futureValue
-      }
 
-      caught shouldBe testGoogleDAO.errorDefinition
+      val errorResponse = response.asInstanceOf[RequestComplete[(StatusCode, ErrorReport)]]
+
+      errorResponse.response match {
+        case (status, errorReport) =>
+          status shouldEqual StatusCodes.InternalServerError
+          errorReport.message should be("Unexpected error during async TSV import")
+      }
     }
 
     "should return error for (async=true) when import service returns sync error" in {
```

Comment:
Test description should be "cwds" now, right?

Reply:
Thanks for noticing this!  This triggered an additional round of find/replace and I _think_ I got them all this time.